src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        ColorSupplier suppler = new ColorSupplier();
+        int ballNumber = random.nextInt(100);
+        String ballColor = suppler.getRandomColor();
+        return new Ball(ballColor,ballNumber);
```

Comment:
There's an unnecessary space before the comma separating the parameters of the `Ball` constructor. According to Java conventions, commas should be followed by a space, but there should be no space before them.